CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - Documentation is now hosted [here](https://py-pdf-parser.readthedocs.io/en/latest/).
+- Font filtering now caches the elements by font. ([#73]https://github.com/jstockwin/py-pdf-parser/pull/73)
```

Comment:
This should be markdown link, i.e. `[text](link)`, inside brackets.
```suggestion
- Font filtering now caches the elements by font. ([#73](https://github.com/jstockwin/py-pdf-parser/pull/73))
```

Reply:
That's what you get when you forget to activate the Markdown plugin in your IDE :man_facepalming: